File: 3rd-party/ssh-client/src/main/java/org/artemir/ssh/client/SSHClient.java
package org.artemir.ssh.client;

final public class SSHClient {
    private SSHClient() {}
    public static Connection connectWithPassword (String host, int port, String user, String pass) {
        UserAuth uAT = new UserAuth(user);
        uAT.setPassword(pass);
        return new Connection(host, port, uAT);
    }
}

File: jphp-ssh-client-ext/src/main/resources/JPHP-INF/sdk/php/ssh/client/SSHClient.php
<?php

namespace php\ssh\client;

/**
 * Class SSHClient
 * @package php\ssh\client
 */
final class SSHClient
{
    private function __construct()
    {
    }

    /**
     * @param string $host
     * @param int $port
     * @param string $user
     * @param string $pass
     * @return Connection
     */
    public static function connectWithPassword($host, $port, $user, $pass) {
    }
}